### DISCUSSION TITLE
Do terms "event handler" and "event listener" have any difference?

### DISCUSSION THREAD

**Discussion by alattalatta** [mdn/mdn-community#612] [Content] [answered]
There are many pages which both terms are present, sometimes even within a single paragraph. IMO it would be better to stick with one to reduce any chance of confusion, but before that, do they differ in their actual meaning, even if little?

"Listening" and "handling" are indeed not equal, but `EventListener` interface defines `handleEvent` method, which... handles events. This also matters in localizing docs: Should I preserve `listener` and `handler`, or is it on me to decide?

**Comment by hamishwillee** [comment, marked as answer]
From MDN

[Event handlers are sometimes called event listeners — they are pretty much interchangeable for our purposes, although strictly speaking, they work together. The listener listens out for the event happening, and the handler is the code that is run in response to it happening](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events)

IMO they should be preserved, unless they are used incorrectly. Most of the time in "Learn" we'll be talking about the handler since that is the code that actually gets run.

**Comment by pepelsbey** [comment]
Closing as answered/settled.
